mkdocs/config/config_options.py

```diff
@@ -465,6 +465,9 @@ def post_validation(self, config, key_name):
             raise ValidationError("The path set in {name}.custom_dir ('{path}') does not exist.".
                                   format(path=theme_config['custom_dir'], name=key_name))
 
+        if 'locale' in theme_config and not isinstance(theme_config['locale'], str):
+            raise ValidationError("'{name}.locale' must be a string.".format(name=theme_config['name']))
```

Comment:
I was a little surprised to find this here. But I suppose it is fine to leave it. I'm curious, what happens if a user sets an invalid value (a language that no translation exists for)? I guess I would expect any error handling to be handled for non-strings there as well.